src/main/resources/templates/fragments/navigation.html

```diff
@@ -0,0 +1,6 @@
+<div th:fragment="navigation">
```

Comment:
This line is required in order to use this component, as you call this fragment name from the page where you needed at